vars/archlinux.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Please add an empty line after all `---`. Unfortunately, I haven't finished [adding this to yamllint](https://github.com/adrienverge/yamllint/pull/393), but that's one check that's missing :smile: